rules/threat_intel/threat_intel_rapid7_threat_command.toml

```diff
@@ -0,0 +1,113 @@
+
+Additional investigation can be done by reviewing the source of the activity and considering the history of the vulnerability that was matched. This can help understand if the activity is related to legitimate behavior.
+
+- Investigation can be validated and reviewed based on the data that was matched and by viewing the source of that activity.
+- Consider the history of the vulnerability that was matched. Has it happened before? Is it happening on multiple machines? These kinds of questions can help understand if the activity is related to legitimate behavior.
+- Consider the user and their role within the company: is this something related to their job or work function?
+"""
+references = [
+    "https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-threatintel.html",
+    "https://docs.elastic.co/integrations/ti_rapid7_threat_command"]
+risk_score = 99
+rule_id = "3a657da0-1df2-11ef-a327-f661ea17fbcc"
+setup = """
+
+## Setup
+
+This rule needs threat intelligence indicators to work.
+Threat intelligence indicators can be collected using an [Elastic Agent integration](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#agent-ti-integration),
+the [Threat Intel module](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#ti-mod-integration),
+or a [custom integration](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#custom-ti-integration).
+
+More information can be found [here](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html).
+
+## Max Signals
+
+This rule is configured to generate more **Max alerts per run** than the default 1000 alerts per run set for all rules. This is to ensure that it captures as many alerts as possible.
+
+**IMPORTANT:** The rule's **Max alerts per run** setting can be superseded by the `xpack.alerting.rules.run.alerts.max` Kibana config setting, which determines the maximum alerts generated by _any_ rule in the Kibana alerting framework. For example, if `xpack.alerting.rules.run.alerts.max` is set to 1000, this rule will still generate no more than 1000 alerts even if its own **Max alerts per run** is set higher.
+
+To make sure this rule can generate as many alerts as it's configured in its own **Max alerts per run** setting, increase the `xpack.alerting.rules.run.alerts.max` system setting accordingly.
+
+**NOTE:** Changing `xpack.alerting.rules.run.alerts.max` is not possible in Serverless projects.
+"""
+severity = "critical"
+tags = [
+    "OS: Windows",
+    "Data Source: Elastic Endgame",
+    "Data Source: Windows",
+    "Data Source: Network",
+    "Data Source: Rapid7 Threat Command",
+    "Rule Type: Indicator Match",
+    "Resources: Investigation Guide",
+    "Use Case: Vulnerability",
+    "Use Case: Asset Visibility",
+    "Use Case: Continuous Monitoring",
+]
+threat_index = ["rapid7-tc-unique-cves"]
```

Comment:
The index is going to be different here. The PR: https://github.com/elastic/integrations/pull/9925 changes this destination index name. Going forwards, this value should be as per transform's index alias name i.e., `logs-ti_rapid7_threat_command_latest.ioc`.

https://github.com/elastic/integrations/blob/3d2e964f3897411a359f9dbc1030820b09f09bc8/packages/ti_rapid7_threat_command/elasticsearch/transform/latest_ioc/transform.yml#L19

Once this rule is available, we will update integration doc (before merging the above PR) to use this rule instead of the custom rule which was shipped along with the package in earlier versions.

Reply:
roger, so new `threat_index` value is `logs-ti_rapid7_threat_command_latest.ioc`?

Reply:
> roger, so new threat_index value is logs-ti_rapid7_threat_command_latest.ioc?

Yes, that is correct.